### PR TITLE
VZ-3989 - Logging in Platform Operator Info to Debug to quiet some noise

### PR DESCRIPTION
**Pull request by bryantidd**
# Description

This PR includes changing several instances of logging at info level to logging at debug level. It is meant to quiet the reconciliation, install, and update messages to reduce noise in the Platform Operator. 

Fixes VZ-9999

# Checklist 

As the author of this PR, I have:

- [X] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/controllers/verrazzano/controller.go [hunk 1]
```diff
@@ -355,7 +355,7 @@ func (r *Reconciler) createClusterRoleBinding(ctx context.Context, log *zap.Suga
 	log.Debugf("Checking if install cluster role binding %s exist", binding.Name)
 	err := r.Get(ctx, types.NamespacedName{Name: binding.Name, Namespace: binding.Namespace}, bindingFound)
 	if err != nil && errors.IsNotFound(err) {
-		log.Infof("Creating install cluster role binding %s", binding.Name)
+		log.Debugf("Creating install cluster role binding %s", binding.Name)
```
Comment (michael-cico) [inline platform-operator/controllers/verrazzano/controller.go:358]: Not sure if we want to mark this Debug?

File: platform-operator/controllers/verrazzano/controller.go [hunk 2]
```diff
@@ -315,7 +315,7 @@ func (r *Reconciler) createServiceAccount(ctx context.Context, log *zap.SugaredL
 	log.Debugf("Checking if install service account %s exist", buildServiceAccountName(vz.Name))
 	err := r.Get(ctx, types.NamespacedName{Name: buildServiceAccountName(vz.Name), Namespace: getInstallNamespace()}, serviceAccountFound)
 	if err != nil && errors.IsNotFound(err) {
-		log.Infof("Creating install service account %s", buildServiceAccountName(vz.Name))
+		log.Debugf("Creating install service account %s", buildServiceAccountName(vz.Name))
```
Comment (michael-cico) [inline platform-operator/controllers/verrazzano/controller.go:318]: Not sure if we want to mark this Debug?

File: platform-operator/controllers/verrazzano/controller.go [hunk 3]
```diff
@@ -480,15 +480,15 @@ func (r *Reconciler) createInstallJob(ctx context.Context, log *zap.SugaredLogge
 	log.Debugf("Checking if install job %s exist", buildInstallJobName(vz.Name))
 	err := r.Get(ctx, types.NamespacedName{Name: buildInstallJobName(vz.Name), Namespace: getInstallNamespace()}, jobFound)
 	if err != nil && errors.IsNotFound(err) {
-		log.Infof("Creating install job %s, dry-run=%v", buildInstallJobName(vz.Name), r.DryRun)
+		log.Debugf("Creating install job %s, dry-run=%v", buildInstallJobName(vz.Name), r.DryRun)
```
Comment (michael-cico) [inline platform-operator/controllers/verrazzano/controller.go:483]: Keeper?

File: platform-operator/controllers/verrazzano/controller.go (hunk 1 above)
Comment (bryantidd) [reply platform-operator/controllers/verrazzano/controller.go:358]: Policy wise? As in its good policy to let them know we are creating a cluster role binding? Or some other reason?

File: platform-operator/controllers/verrazzano/controller.go [hunk 4]
```diff
@@ -1333,7 +1333,7 @@ func (r *Reconciler) watchJobs(namespace string, name string, log *zap.SugaredLo
 	if err != nil {
 		return err
 	}
-	log.Infof("Watching for jobs to activate reconcile for Verrrazzano CR %s/%s", namespace, name)
+	log.Debugf("Watching for jobs to activate reconcile for Verrrazzano CR %s/%s", namespace, name)
```
Comment (michael-cico) [inline platform-operator/controllers/verrazzano/controller.go:1336]: Typo here, `Verrrazzano` --> `Verrazzano`

Comment (bryantidd) [reply platform-operator/controllers/verrazzano/controller.go:1336]: I guess I should fix that....  I just had changed it to Debug, but fixing what's broken while your there is important

File: platform-operator/controllers/verrazzano/controller.go (hunk 2 above)
Comment (bryantidd) [reply platform-operator/controllers/verrazzano/controller.go:318]: Policy wise?

File: platform-operator/controllers/verrazzano/upgrade.go [hunk 5]
```diff
@@ -47,7 +47,7 @@ func (r *Reconciler) reconcileUpgrade(log *zap.SugaredLogger, cr *installv1alpha
 			return newRequeueWithDelay(), err
 		}
 		if !installed {
-			log.Infof("Skip upgrade for %s, not installed", comp.Name())
+			log.Debugf("Skip upgrade for %s, not installed", comp.Name())
```
Comment (pfmackin) [inline platform-operator/controllers/verrazzano/upgrade.go:50]: This might be good to keep as Infof

**Review by michael-cico — APPROVED** [review]

**Review by pfmackin — DISMISSED** [review]

**Review by pfmackin — DISMISSED** [review]

File: platform-operator/controllers/verrazzano/controller.go (hunk 2 above)
Comment (michael-cico) [reply platform-operator/controllers/verrazzano/controller.go:318]: as you can see I changed my mind on this one....I deleted the original comment.

File: platform-operator/controllers/verrazzano/controller.go (hunk 1 above)
Comment (michael-cico) [reply platform-operator/controllers/verrazzano/controller.go]: like above, changed my mind before you even replied...

**Review by michael-cico — DISMISSED** [review]

**Review by markxnelson — APPROVED** [review]

**Pull request merged**